Website/src/items/templates/edgewebsite/PaymentProgress/__Standard Values.yml

```diff
@@ -49,12 +49,6 @@ SharedFields:
           id="{9E28A6C1-7D05-5C7F-81AF-1AF94C7A2F4D}"
           par=""
           ph="/jss-main" />
-        <r
```

Comment:
`PaymentProgress` is a datasource template (not a page template). It should not have a layout and renderings. Could you please remove its layout and renderings? Thanks